docs/get-started/lotus/chain.md

```diff
@@ -50,12 +50,25 @@ These lightweight state snapshots **do not contain any message receipts**. To ge
     lotus daemon --import-snapshot minimal_finality_stateroots_latest.car
     ```
 
-You can skip the `sha256sum` check and use the snapshot URL directly if you'd prefer:
+We strongly recommend you verify the checksum of the download. However, you can skip the `sha256sum` check and use the snapshot URL directly if you'd prefer:
 
 ```bash
 lotus daemon --import-snapshot https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/minimal_finality_stateroots_latest.car
 ```
 
+#### Get message receipts
```

Comment:
I think this is getting things more confusing overall. My suggestion is to not mention receipts here at all.

Reply:
Sorry, wasn't clear enough. Receipts are not something that is included in snapshots, period: https://github.com/filecoin-project/lotus/issues/3849

What I am saying is: let's not talk about receipts *at all*. They are an advanced thing that a very small part of the population care about...

Reply:
Ok, follow up questions: 

1. Both snapshots contain messages, but neither snapshot contains message receipts. Correct?
2. The best and easiest way to get message receipts is **not** use any snapshots and to just sync the chain normally like a pleb. Correct?